.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:10.2
```

Comment:
Should be the latest version. For what it's worth, we probably don't need CI on all these repos.

Reply:
It'll be overhead for maintaining the configurations and updating nodejs versions that probably doesn't save that much. Tests should be cloned and run anyways.

Reply:
We would want to run with many different nodejs version also, but that's probably something for another day.